spec/requests/v1/ont/runs_spec.rb

```diff
@@ -148,28 +187,185 @@ def json
         expect(fc4.position).to eq(attr4[:position])
         expect(fc4.ont_pool_id).to eq(attr4[:ont_pool_id])
       end
+
+      context 'messages' do
+        let(:run) { create(:ont_gridion_run, flowcell_count: 3) }
+
+        let(:body) do
+          {
+            data: {
+              type: 'runs',
+              id: run.id,
+              attributes: {
+                ont_instrument_id: run.ont_instrument_id,
+                state: 'completed'
+              }
```

Comment:
Do you want to test the run creation with any flowcell data too maybe. Would this flowcell data also be sent to the MLWH?